proxy_docker/app/script/requesthandler.sh

```diff
@@ -408,6 +408,38 @@ main() {
           response_to_client "${response}" ${?}
           break
           ;;
+        ln_listpeers)
+          # GET http://192.168.111.152:8080/ln_listpeers
+
+          response=$(ln_listpeers)
+          response_to_client "${response}" ${?}
+          break
+          ;;
+        ln_listfunds)
+          # GET http://192.168.111.152:8080/ln_listfunds
+          response=$(ln_listfunds)
+          response_to_client "${response}" ${?}
+          break
+          ;;
+        ln_listpays)
+          # GET http://192.168.111.152:8080/ln_listpays
+	  response=$(ln_listpays)
+          response_to_client "${response}" ${?}
+          break
+          ;;
+        ln_getroute)
+          # GET http://192.168.111.152:8080/ln_getroute/<node_id>/<msatoshi>/<riskfactor>
+	  response=$(ln_getroute $(echo "${line}" | cut -d ' ' -f2 | cut -d '/' -f3) $(echo "${line}" | cut -d ' ' -f2 | cut -d '/' -f4) $(echo "${line}" | cut -d ' ' -f2 | cut -d '/' -f5))
+          response_to_client "${response}" ${?}
+          break
+          ;;
+        ln_withdraw)
```

Comment:
I'd love to see a `"all":true` in the args, so it's simple to just withdraw everything.  In this case, `"satoshi"` is not required.

Reply:
`satoshi` is required for route finding. https://lightning.readthedocs.io/lightning-getroute.7.html

Reply:
No, I mean add a property like this: `{...,"all":true,...}` so it withdraws everything and takes care of the fees and all.  In this case, the `"satoshi"` property is obviously not required, you would take it from the c-lightning on-chain balance.

Reply:
from c-lightning docs:
`satoshi is the amount to be withdrawn fr..... *The string all can be used to specify withdrawal of all available funds. Otherwise, it is in satoshi precision;* ......`
So if i understand you correctly, the default would be to `satoshi:all` to withdraw all non used outputs ?

Reply:
I would say it's cleaner and more concise to keep `satoshi:all` but I've added the `all` flag as requested:
```bash
if [ "${all}" == true ] || [ "${all}" == "true" ] ; then
        satoshi="all"
fi 
```

Reply:
I was expecting that to be a problem since satoshi is a number but "all" is a string...